sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -16,6 +16,10 @@ import { CanonicalCode } from "@azure/core-tracing";
 import { AuthenticationError, AuthenticationErrorName } from "./errors";
 import { createSpan } from "../util/tracing";
 
+
+import { createClientLogger } from "@azure/logger";
+const logger = createClientLogger("IdentityClient");
```

Comment:
To follow the [pattern we're using elsewhere](https://github.com/Azure/azure-sdk-for-js/pull/5605/commits/d28c9bf351589ffae3240eeaf24dd7f50ff03702#diff-b7f79eb11bce1239d2cc0fea068e4086R29), I'd use `identity` here

Reply:
Done. although I rebased so now we're not able to prove it. But I swear I did it.